views/layouts/layout.ejs

```diff
@@ -1,13 +1,20 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" type="text/css" href="/styles/lvlOfKnowledge.css">
+    <link rel="stylesheet" type="text/css" href="/styles/tagsList.css">
```

Comment:
Although not wrong, maybe you you'd want to keep CSS rules in on file, for now. Say `styles.css`.

Reply:
Exactly. `styles.css` should do for now. If you're using SASS/SCSS - and you have a more complex application, then you will be splitting CSS, in a logical manner, into multiple files.

Reply:
Ok I will do that. I was thinking to split them untill they are actually done but it doesn't make sense